src/schema/v2/me/sale_registrations.ts

```diff
@@ -38,24 +38,21 @@ export const SaleRegistrationConnection: GraphQLFieldConfig<
   resolve: async (
     _root,
     { isAuction: is_auction, live, published, sort, ...paginationArgs },
-    { meBiddersLoader, salesLoader }
+    { meBiddersLoader, salesLoaderWithHeaders }
   ) => {
     if (!meBiddersLoader) return null
     const { page, size, offset } = convertConnectionArgsToGravityArgs(
       paginationArgs
     )
-    const { body: sales, headers } = (await (salesLoader(
-      {
-        is_auction,
-        live,
-        published,
-        sort,
-        page,
-        size,
-        total_count: true,
-      },
-      { headers: true }
-    ) as any)) as BodyAndHeaders
+    const { body: sales, headers } = (await (salesLoaderWithHeaders({
+      is_auction,
+      live,
+      published,
+      sort,
+      page,
+      size,
+      total_count: true,
+    }) as any)) as BodyAndHeaders
```

Comment:
This form is pretty confusing to read, can we type it like:

```tsx
const response: BodyAndHeaders = await salesLoaderWithHeaders(...)
const { body, headers } = response
...
```

Reply:
Yessir!